modules/client/src/main/kotlin/jp/co/soramitsu/iroha2/TransactionBuilder.kt

```diff
@@ -26,13 +26,15 @@ class TransactionBuilder(builder: TransactionBuilder.() -> Unit = {}) {
     val instructions: Lazy<ArrayList<Instruction>>
     var creationTimeMillis: BigInteger?
     var timeToLiveMillis: BigInteger?
+    var nonce: Long?
     var metadata: Lazy<HashMap<String, Value>>
 
     init {
         accountId = null
         instructions = lazy { ArrayList() }
         creationTimeMillis = null
         timeToLiveMillis = null
+        nonce = null
```

Comment:
Let make nonce a random number by default. This approach is recommended by Iroha2 team